src/lib/lessWatchCompilerUtils.js

```diff
@@ -117,6 +114,35 @@ define(function (require) {
       };
 
     },
+    resolveOutputPath: function (filePath) {
```

Comment:
I would love a test for this function and remove references to `lessWatchCompilerUtilsModule`... but it's not consistent everywhere else anyways.